server/uptime-calculator.js

```diff
@@ -543,7 +550,9 @@ class UptimeCalculator {
         if (type === "minute" && num > 24 * 60) {
             throw new Error("The maximum number of minutes is 1440");
         }
-
+        if (type === "day" && num > 365) {
```

Comment:
Have not had a deeper look at this code: what happens if it is?

Reply:
I also don't know what will happen if the minute exceeds 1440. I think there should be a limit here for day.

Reply:
true, the limit defined for this is 365. Good catch. 👍🏻

For others:
https://github.com/louislam/uptime-kuma/blob/6e30f71947f9d9af1bde7158533cd822d0b5b37d/server/uptime-calculator.js#L48